ietf/draft-ietf-calext-jscalendar-icalendar-09.txt

```diff
@@ -0,0 +1,6272 @@
+         object on which the iCalComponent property is set on.  The
+         value for each key contains information about the iCalendar
+         property which converted to the object property located at that
+         key (see Section 5.1.3).
+
+         The key MUST be a valid key of a PatchObject as defined in
+         Section 1.4.9 of [RFC8984].  The key MUST NOT point into a
+         nested property, unless there is no way to otherwise preserve
+         the iCalendar property elements.  For example, if a VEVENT
+         contains an ATTENDEE property, then any iCalendar information
+         about the ATTENDEE property MUST be set in the iCalProperty
+
+
+
+Stepanek                   Expires 16 May 2025                [Page 100]
+
+Internet-Draft                 JSCalendar                  November 2024
+
+
+         property of the converted Participant object (see
+         Section 2.3.4).  In contrast, information about a RDATE
+         property contained in a VEVENT would be located by a key
+         pointing into the recurrenceOverrides property of the Event
+         object (see Section 2.3.35).
+
+      properties: *[][] (optional)  This contains properties of the
+         iCalendar component.  The value MUST be a list of iCalendar
+         properties formatted in jCal as defined in Section 3.4 of
+         [RFC7265].
+
+5.1.3.  iCalProperty
+
+   Name:
+      iCalProperty
+
+   Context:
+      Any JSCalendar object
+
+   Type:
+      ICalProperty (optional)
+
+   Description:
+      This contains conversion-related information about an iCalendar
+      property that got partially or fully converted to JSCalendar.  It
+      allows to preserve the name of the iCalendar property and some or
+      all of its parameters.
+
+      An ICalProperty object has the following properties:
+
+      @type: String (mandatory)  This specifies the type of this object.
+         This MUST be ICalProperty.
+
+      name: String (mandatory)  This is the name of the iCalendar
+         property in lowercase.
+
+      valueType: String (optional)  This is the name of the iCalendar
+         property value type in lowercase.
+
+      parameters: String[String] (optional)  This contains parameters of
+         the iCalendar property.  The value MUST comply with iCalendar
+         parameters formatted in jCal as defined in Section 3.5 of
+         [RFC7265].
+
+5.1.4.  locations (Participant)
+
+   Name:
+      locations
+
+
+
+Stepanek                   Expires 16 May 2025                [Page 101]
+
+Internet-Draft                 JSCalendar                  November 2024
+
+
+   Context:
+      Participant
+
+   Type:
+      Id[Location] (optional)
+
+   Description:
+      This represents locations associated with the participant.  In
+      contrast to the locationId property, this allows to associate
+      multiple locations with the Participant object, and to express
+      that a location just is relevant for this participant.
```

Comment:
"just is relevant" is weird. Maybe "... location is only relevant for this participant" ?